docs/process.rst

```diff
@@ -86,10 +89,54 @@ pipeline script.
 .. warning:: Since Nextflow uses the same Bash syntax for variable substitutions in strings, you need to manage them
   carefully depending on if you want to evaluate a variable in the Nextflow context - or - in the Bash environment execution.
 
-When you need to access a system environment variable  in your script you have two options. The first choice is as
-easy as defining your script block by using a single-quote string. For example::
+When you need to access a system's environment variable in your script you have three options. 
+
+The first choise is to use Nexflow and Bash environment variables simultaneously. 
+
+Nextflow will use Bash's environment variables unless they are overwritten within the Nextflow context. 
+The first example will print `Nextflow`::
+
+    PATH = "Nextflow"
+    process printNextflow {
+       echo true
+
+       """"
+       echo The path is: $PATH
+       """"
+
+    }
+
+While the second example will print Bash's ``PATH`` variable::
```

Comment:
More than Bash's variable I think it would be better to say "the ``PATH`` variable defined in the system environment".